crates/autopilot/src/solvable_orders.rs

```diff
@@ -588,153 +542,6 @@ mod tests {
         assert_eq!(orders_, orders[1..]);
     }
 
-    #[tokio::test]
-    async fn caches_orders_and_balances() {
-        let mut balance_fetcher = MockBalanceFetching::new();
```

Comment:
Removed because it tests the data that used to be shared with Orderbook which is no longer the case.